config/manager/manager.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    control-plane: terraform-k8s
+  name: system
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: operator
+  namespace: system
+  labels:
+    control-plane: terraform-k8s
+spec:
+  selector:
+    matchLabels:
+      control-plane: terraform-k8s
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        control-plane: terraform-k8s
+    spec:
+      containers:
+      - command:
+        - /terraform-k8s
+        args:
+        - --enable-leader-election
+        image: terraform-k8s:latest
+        imagePullPolicy: IfNotPresent
+        name: terraform-k8s
+        env:
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: POD_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: OPERATOR_NAME
+              value: "terraform-k8s"
+            - name: TF_CLI_CONFIG_FILE
+              value: "/etc/terraform/.terraformrc"
```

Comment:
We also need to add `TF_URL` here, so that Terraform Enterprise is supported. The helm deployment has it listed like this:

```
            - name: TF_URL
              value: "{{ default "" .Values.syncWorkspace.tfeAddress }}"
```

I'm not sure what the kustomize equivalent is offhand, but I found a blog post that has an [example](https://blog.stack-labs.com/code/kustomize-101/) of adding an environment variable like this. They use an overlay. I suppose TFE would be one overlay, so the env var can be injected if the user is using TFE.

CC @redeux because this could help us get rid of terraform-helm, if you wanted to. I just wanted to bring up the idea since we've had some trouble keeping the terraform-helm repo in sync with the operator's repo. I think it would be simpler to use kustomize instead, since it's built into this new operator version.